features/tbl-row-props.feature

```diff
@@ -6,20 +6,33 @@ Feature: Get and set table row properties
 
   Scenario Outline: Get row height
     Given a table row having a height of <height-setting>
-     Then the reported height of the row is <reported-height>
+     Then the reported height rule of the row is <reported-height-rule>
+      And the reported height value of the row is <reported-height>
 
     Examples: Table row height settings
-      | height-setting      | reported-height |
-      | no explicit setting | None            |
-      | 3 inches            | 3 inches        |
+      | height-setting      | reported-height-rule | reported-height |
+      | no explicit setting | None                 | None            |
+      | automatic height    | AUTO                 | 0               |
+      | at least 2 inches   | AT_LEAST             | 2 inches        |
+      | exactly 3 inches    | EXACT                | 3 inches        |
 
 
   Scenario Outline: Set row height
     Given a table row having a height of <height-setting>
      When I set the row height to <new-setting>
-     Then the reported height of the row is <reported-height>
+     Then the reported height rule of the row is <reported-height-rule>
+      And the reported height value of the row is <reported-height>
 
     Examples: table column height values
-      | height-setting      | new-setting | reported-height |
-      | no explicit setting | 3 inches    | 3 inches        |
-      | 4 inches            | 3 inches    | 3 inches        |
+      | height-setting      | new-setting       | reported-height-rule | reported-height |
+      | no explicit setting | automatic height  | AUTO                 | 0               |
+      | at least 2 inches   | automatic height  | AUTO                 | 0               |
+      | exactly 4 inches    | automatic height  | AUTO                 | 0               |
+      | no explicit setting | at least 3 inches | AT_LEAST             | 3 inches        |
+      | automatic height    | at least 3 inches | AT_LEAST             | 3 inches        |
+      | at least 2 inches   | at least 3 inches | AT_LEAST             | 3 inches        |
+      | exactly 4 inches    | at least 3 inches | AT_LEAST             | 3 inches        |
+      | no explicit setting | exactly 5 inches  | EXACT                | 5 inches        |
+      | automatic height    | exactly 5 inches  | EXACT                | 5 inches        |
+      | at least 2 inches   | exactly 5 inches  | EXACT                | 5 inches        |
+      | exactly 4 inches    | exactly 5 inches  | EXACT                | 5 inches        |
```

Comment:
Hi @wich, I've pushed a refactored .feature file. Next step will be for you to get the steps etc. working with it. Here is a summary of the changes I made:

* acceptance tests for Row.height are separate from those for Row.height_rule. This is the main thing. This makes them more compact and independent, the general principle being that tests should test one thing only. The practical aspect is you don't have to test all the combinations and permutations, so they're also shorter. And of course they're easier to read and reason about.

* The corpus of acceptance tests embody a (my) learning curve in which I've refined the style to a compact and direct wording. So unfortunately not all the acceptance tests are as good an example as others :)  This gives an example of the current style. I haven't gone back to update the existing ones, maybe one day :)

If you can adjust the steps and the .pptx test file to work with the new feature file we'll get this committed and move on from there.